lib/module.js

```diff
@@ -57,7 +57,7 @@ module.exports = function (moduleOptions) {
 
   // Add vue-apollo and apollo-client in common bundle
   if (!isNuxtVersion2) {
-    this.addVendor(['vue-apollo', 'js-cookie', 'cookie'])
+    this.addVendor(['vue-apollo', 'universal-cookie'])
```

Comment:
I guess it should be `cookie-universal` :)

```suggestion
    this.addVendor(['vue-apollo', 'cookie-universal'])
```

Reply:
Yeah, my bad.